Tests/UriUnitTests/UriUnitTests.cs

```diff
@@ -0,0 +1,716 @@
+using nanoFramework.TestFramework;
+using System;
+
+namespace System.Http
+{
+    [TestClass]
+    public class UriUnitTests
+    {
+        Uri uri;
```

Comment:
I know those are "just" tests, but better if you specify the visibility. I guess private will work perfectly

Reply:
I'm afraid they have to be public methods for the test discoverer to find them...

Reply:
It was about the visibility of Uri, the variables, not the methods :-)